build/Dockerfile.ci

```diff
@@ -65,7 +65,7 @@ WORKDIR /build/windows-machine-config-operator
 COPY build build
 COPY main.go .
 COPY controllers controllers
-COPY deploy deploy
+COPY bundle bundle
```

Comment:
don't we need to make this change in the developer Dockerfile as well?

Reply:
@mansikulkarni96 you mean in `build/Dockerfile`?
The deploy directory wasnt copied as it is only needed for CI, so there is nothing to correct